python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -432,33 +374,42 @@ async def run_cwfs(self):
                 f"Running cwfs in " f"{self.intra_visit_id}/{self.extra_visit_id}."
             )
 
-        self.intra_exposure = await loop.run_in_executor(
-            executor, self.get_isr_exposure, self.intra_visit_id
+        self.intra_exposure, self.extra_exposure = await asyncio.gather(
+            get_image(parse_visit_id(self.intra_visit_id)),
+            get_image(parse_visit_id(self.extra_visit_id)),
         )
 
-        # create a clone to be used for detection
-        self.detection_exp = self.intra_exposure.clone()
+        # Run source detection - FIXME: make this run concurrently
+        self.log.debug("Running source detection")
 
-        self.extra_exposure = await loop.run_in_executor(
-            executor, self.get_isr_exposure, self.extra_visit_id
+        self.intra_result = await loop.run_in_executor(
+            executor,
+            functools.partial(
+                self.qm.run, self.intra_exposure, donutDiameter=2 * self.side
+            ),
         )
+        self.extra_result = await loop.run_in_executor(
```

Comment:
What does the `self.qm.run` do? is it doing isr plus source detection? If it is doing the source selection, why do it on both? Wouldn't it be best to do the source selection on an image that is intra + extra, the way we did before?

Reply:
It does just the source finding. The ISR is handled in `get_image`
The reason I switched away from adding the two together is mostly for simplicity. The source selection (if run in parallel) is pretty quick and doesn't add time. 
There is also the effect that simply adding the doughnuts together (which are shifted a bit) makes a weird shape... although I don't think this really matters.

Reply:
Discussed in person. Will perform two source finding algorithms as there is a concern about if the algorithm works on combined donuts (which may be odd shapes).
Will add a check to make sure the results are within ~100 pixels of each other.